### PR TITLE
WIP Adding Tests

### PR DESCRIPTION
**Pull request by aps120797**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -24,21 +25,26 @@ module.exports = (app) => {
...
+    const { owner, repo } = context.repo()
     const headDate = getDate.headDate()
     const tailDate = getDate.tailDate()
-    let config = await getConfig(context, 'weekly-digest.yml')
-    config = fixConfig(config)
-    console.log(`Publish Day: ${getNumDayFromLongDay(config.publishDay)}, Today: ${moment().day()}`)
-    if (moment().day() === getNumDayFromLongDay(config.publishDay)) {
-      const { owner, repo } = context.repo()
-      weeklyDigest(context, {owner, repo, headDate, tailDate}, config)
+    let hasDuplicates = checkDuplicates(context, {owner, repo, headDate})
```
Comment (gr2m) [inline index.js]: I think you need `await checkDuplicates()` here?

File: index.js [hunk 2]
```diff
@@ -24,21 +25,26 @@ module.exports = (app) => {
...
+    if (hasDuplicates) {
+      console.log(`Weekly Digest for this week has already been published for ${owner}/${repo}`)
```
Comment (gr2m) [inline index.js]: maybe you could log the full URL to the issue that was already created?

Comment (aps120797) [reply index.js]: :+1: Doing so.

Comment (aps120797) [reply index.js]: Fixed. 😄

File: index.js (hunk 1 above)
Comment (aps120797) [reply index.js]: Missed that. Fixed it. 😃

**Pull request merged**